src/memray/reporters/templates/__init__.py

```diff
@@ -28,7 +28,11 @@ def include_file(name: str) -> Markup:
     return env
 
 
-def get_report_title(*, kind: str, show_memory_leaks: bool) -> str:
+def get_report_title(
+    *, kind: str, show_memory_leaks: bool, inverted: bool = False
+) -> str:
+    if inverted and kind == "flamegraph":
+        kind = f"inverted {kind}"
```

Comment:
Don't we want to return "inverted flamegraph report (memory leaks)" if `show_memory_leaks` is true? I think what we want is:
```py
    parts = []
    if inverted:
        parts.append("inverted")
    parts.append(kind)
    parts.append("report")
    if show_memory_leaks:
        parts.append("(memory leaks)")
    return " ".join(parts)
```

Also: Don't worry about doing the `and kind == "flamegraph"` here. I think it's fine to return "inverted table report" if someone calls this with `kind="table"`. We know they won't, but if they for some reason do, that would be sane behavior, so there's no reason to avoid it.